velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,25 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+HashBuild, HashAggregation
+------------------
+.. list-table::
```

Comment:
nit: add empty line between the header and this table

Reply:
Fixed